docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/link/link.twig

```diff
@@ -36,18 +35,32 @@
   {% endset %}
 {% endif %}
 
+{% if is_external %}
+  {% set external_icon %}
+    {% include '@atoms/icon/icon.twig' with {
+       symbol: 'upper-right-arrow',
+     } only %}
+  {% endset %}
+{% endif %}
+
 {% set link_text %}
   {% if icon and button_icon %}
     {% if icon_placement == 'before' %}
-      {{ button_icon }}{{ text|raw }}
+      {{ button_icon }}{{ text|raw }} {{ external_icon }}
     {% elseif icon_placement == 'after' %}
-      {{ text|raw }}{{ button_icon }}
+      {{ text|raw }}{{ external_icon }}{{ button_icon }}
```

Comment:
this should be the other way around - similar to button

<img width="220" alt="CivicTheme__Design_System_v1_0_0__WIP_" src="https://user-images.githubusercontent.com/378794/188518977-d708e861-2751-4500-92da-6b2998606a8e.png">

Reply:
please rename `button_icon` - there is no button anymore

Reply:
![Screenshot 2022-09-06 at 8 39 04 AM](https://user-images.githubusercontent.com/83997348/188538941-52e693cf-dee6-4039-92ac-0ec188b09a13.png)
![Screenshot 2022-09-06 at 8 39 44 AM](https://user-images.githubusercontent.com/83997348/188539014-486b9002-5519-4b7f-8a67-ed0893ac0006.png)